server/src/main/java/timely/api/model/Metric.java

```diff
@@ -34,10 +34,23 @@
 @WebSocket(operation = "put")
 public class Metric implements TcpRequest, HttpPostRequest, WebSocketRequest {
 
+    private static final ThreadLocal<ByteBuffer> WRITE_BUFFER = new ThreadLocal<ByteBuffer>() {
```

Comment:
Curious....is the cost of getting a ByteBuffer out of the thread pool synchronized map and then clearing it cheaper than simply allocating another ByteBuffer?